cfme/tests/configure/test_server_name.py

```diff
@@ -23,27 +24,29 @@ def test_server_name(request, appliance):
 
     view = navigate_to(appliance.server, 'Details')
     old_server_name = view.server.basic_information.appliance_name.read()
+    assert old_server_name != ''  # name should at least be set
 
     @request.addfinalizer
     def _ensure_name_reset():
-        appliance.server.settings.update_basic_information({'appliance_name': old_server_name})
-
-    new_server_name = "{}-TEST".format(old_server_name)
-    appliance.server.settings.update_basic_information({'appliance_name': new_server_name})
-    flash_message = (
-        'Configuration settings saved for {} Server "{} [{}]" in Zone "{}"'.format(
-            appliance.product_name,
-            appliance.server.name,
-            appliance.server.sid,
-            appliance.server.zone.name))
-
-    view.flash.assert_message(flash_message)
+        appliance.rename(old_server_name)
+
+    new_server_name = "RENAME-TEST"
+    assert view.server.basic_information.appliance_name.fill(new_server_name)
+    assert view.server.save.is_enabled
+    view.server.save.click()  # no boolean return
+    assert appliance.server.name == new_server_name
+
+    view.flash.assert_success_message(
+        'Configuration settings saved for {} Server "{} [{}]" in Zone "{}"'
```

Comment:
Optional: Can we not make format this using keys?

Reply:
I would gladly merge a PR that changes this :) (already merged by the time I saw this)